dist/images/install.sh

```diff
@@ -3408,6 +3409,20 @@ reload(){
   kubectl rollout status deployment/kube-ovn-monitor -n kube-system
 }
 
+env-check(){
+  set +e
+
+  KUBE_OVN_NS=kube-system
+  podNames=`kubectl get pod --no-headers -n $KUBE_OVN_NS | grep kube-ovn-cni | awk '{print $1}'`
+  for pod in $podNames
+  do
+    echo "****************************************************"
+    echo "Start environment check in pod $pod"
```

Comment:
Would be better to print `Start environment check for node $node`.

Reply:
modified